### ISSUE TITLE
Installing azurerm plugin failed - GitHub API rate limit exceeded

### ISSUE THREAD

**Issue by lkurzyniec** [terraform-linters/tflint#1655]
### Summary

Would it be possible that `tflint` download plugins in other way than through API which is rate limited?

### Command

tflint --config=$(pwd)/.tflint.hcl --init

### Terraform Configuration

```terraform
not relevant
```

### TFLint Configuration

```HCL
plugin "terraform" {
  enabled = true
  preset  = "recommended"
}

plugin "azurerm" {
  enabled = true
  version = "0.20.0"
  source  = "github.com/terraform-linters/tflint-ruleset-azurerm"
}
```

### Output

```Shell
TFLint version 0.44.1
Installing `azurerm` plugin...
Failed to install a plugin; Failed to download tflint-ruleset-azurerm_linux_amd64.zip: GET https://api.github.com/repos/terraform-linters/tflint-ruleset-azurerm/releases/assets/87802080: 403 API rate limit exceeded for 10.xx.xxx.xx. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) [rate reset in 26s]
##[error]Bash exited with code '1'.
```

### TFLint Version

0.44.1

### Terraform Version

1.3.5

### Operating System

- [X] Linux
- [ ] macOS
- [ ] Windows
